### DISCUSSION TITLE
Export map as standalone package

### DISCUSSION THREAD

**Discussion by xisoP2021** [carla-simulator/carla#6291] [Q&A]
Hi all,
   Carla version: 0.9.14 (source);  OS: Ubuntu18.04
   I exported a RoadRunner example map <T_intersection> and tried to ingest the map in Carla. After some customizations, I wanted to export the new map as a standalone package for distribution. I had the following problems:

-    For ingestion I used "**make import  ARGS="--package=T_intersection**". I found a number of materal/texture  were lost during the process. Instead I used RoadRunner Plugin to ingest the map, all details were preserved. The difference was quite obvious:
![make_import](https://user-images.githubusercontent.com/90833358/223628256-3d94ccb1-77ea-46f4-bb35-af45f253f026.PNG)
![roadrunner_plugin](https://user-images.githubusercontent.com/90833358/223628276-d0d7193a-2d2d-43b8-8926-a99477cfd2a7.PNG)

-    However this way the map was not recognized as a map package, therefore cannot be exported using "**make package**". I tried several methods including manually creating .json file in the map root diretory but that didn't work.
-    Another way I tried is to created a new folder under an existing map package (the one automatically created using make import argument): T_intersection/pluginImport, in which a map was added using RoadRunner Plugin. Then run "**make package ARGS="--packages=T_intersection**" hoping that Carla would export both maps as package (since they were both in the T_intersection directory). It failed cause the Plugin Map was not recognized as inside the "T_intersection" map package.

Is there any way to export maps that are imported using RoadRunner Plugin? Here's the link of my .fbx and .xodr file:
https://drive.google.com/file/d/1qj6yt6aKwTdzKrV_IL3txkKYsNvWVgG2/view?usp=sharing

Thanks in advance

**Comment by rahulxtpbg** [comment]
Did you ever manage to resolve this? Also, in my case, despite using the RoadRunner plugin to import/ingest the map, it still almost looks the same as an import without the plugin. Did you also run into this issue?
